configuration/azure/appconfig/appconfig.go

```diff
@@ -248,9 +266,103 @@ func (r *ConfigurationStore) getLabelFromMetadata(metadata map[string]string) *s
 }
 
 func (r *ConfigurationStore) Subscribe(ctx context.Context, req *configuration.SubscribeRequest, handler configuration.UpdateHandler) (string, error) {
-	return "", fmt.Errorf("Subscribe is not implemented by this configuration store")
+	subscribeID := uuid.New().String()
```

Comment:
This may panic,  see https://pkg.go.dev/github.com/google/uuid#New

Please replace with `NewRandom` and handle the error.

Reply:
@shubham1172 I think it will make sense to have retry in case of duplicate string rather than failing the subscribe. Given that probability of duplicate is very rare will this matter much. Thoughts ?

quote from new Random method - 
_//  Randomly generated UUIDs have 122 random bits.  One's annual risk of being
//  hit by a meteorite is estimated to be one chance in 17 billion, that
//  means the probability is about 0.00000000006 (6 × 10−11),
//  equivalent to the odds of creating a few tens of trillions of UUIDs in a
//  year and having one duplicate._

Reply:
I think it is safe to avoid duplication handling if these UUIDs are purely in-memory and ephemeral.